api/numaresourcesoperator/v1/numaresourcesoperator_defaults.go

```diff
@@ -38,6 +38,9 @@ func (ngc *NodeGroupConfig) Default() {
 	if ngc.InfoRefreshMode == nil {
 		ngc.InfoRefreshMode = defaultInfoRefreshMode()
 	}
+	if ngc.RTEPause == nil {
```

Comment:
(here and everywhere) I'm still contemplating if we should rename `RTEPause` to `InfoRefreshPause` or not. What do you think?

Reply:
yep, agree. I had several options in mind ( nrtDataPublish/ nrtUpdatingMode/ rteFunctionalityMode/ NRTInfoSuppress) but InfoRefreshPause sounds better as it goes along with the rest of the other fields under the ngc.